python/lsst/ts/m2gui/model.py

```diff
@@ -291,12 +297,17 @@ def go_to_position(self, x, y, z, rx, ry, rz):
             Rotation y in arcsec.
         rz : `float`
             Rotation z in arcsec.
+
+        Raises
+        ------
+        `RuntimeError`
+            Not in the closed-loop control.
         """
 
         if self.is_enabled_and_closed_loop_control():
             self.log.info(f"Move to the position: ({x}, {y}, {z}, {rx}, {ry}, {rz}).")
         else:
-            self.log.info("This can only be done in the closed-loop control.")
+            raise RuntimeError("This can only be done in the closed-loop control.")
```

Comment:
Done in the commit: [b2dae7c](https://github.com/lsst-ts/ts_m2gui/pull/10/commits/b2dae7c31b0f66668b2974b6761c5d7795e35f4b). I did not log it because I think that is just a duplication of information. Thanks!

Reply:
Well obviously you are doing that to log what happened. Once time will come with observer complaining "the position command wasn't executed", you will have proof what happened (e.g. why answered).